### ISSUE TITLE
Improved header navbar

### ISSUE THREAD

**Issue by oceanByte** [learn-ocean/ocean-academy#29]
The current navbar is not suited to contain more than four items (depending on font size and string length). We expect more items in the near future that we need to place in the navbar. To enable this we need to rethink the current navbar layout. 

We need a solution for links like "terms" and "about us" which you need to reach on every page. Maybe something like: 

Logged out user:
About us | Terms | Login  

Logged in user: 
About us | Terms | Logout (dropdown, see image))

I suggest following a common practice and have a dropdown menu that contains all the links similar to the current style on github (logged in user):
![image](https://user-images.githubusercontent.com/75223088/101176358-a774ea80-3646-11eb-960e-81040156f3a0.png)

**Comment by SarahKay99** [comment]
Can we consider this closed? Or are there improvements that still need to be made

**Comment by oceanByte** [comment]
> Can we consider this closed? Or are there improvements that still need to be made

We will close it once the given feature is merged.

**Comment by SarahKay99** [comment]
> We will close it once the given feature is merged.
I thought it was merged, with the hamburger menu thing? I am probably misunderstanding the task.

**Comment by oceanByte** [comment]
> I thought it was merged, with the hamburger menu thing? I am probably misunderstanding the task.

Is the hamburger menu triggered by the width of the viewport or by the number of elements in the navbar? Because we might have more than four entries in the header. If we can have more than four entries with the current implementation I would move this issue outside of the shrimp sprint. 

But in general, we will need a more compelling solution to this problem. A second hamburger menu is an unusual design pattern and might lead to the impression that something is not working as intended.

**Comment by SarahKay99** [comment]
> Is the hamburger menu triggered by the width of the viewport or by the number of elements in the navbar? Because we might have more than four entries in the header. If we can have more than four entries with the current implementation I would move this issue outside of the shrimp sprint.
> 
> But in general, we will need a more compelling solution to this problem. A second hamburger menu is an unusual design pattern and might lead to the impression that something is not working as intended.

The hamburger is triggered by the width of the viewport. So I'm pretty sure we could add any number of elements and it would still work. Maybe we should move this outside of Shrimp and make it look less unusual later

**Comment by oceanByte** [comment]
> The hamburger is triggered by the width of the viewport. So I'm pretty sure we could add any number of elements and it would still work. Maybe we should move this outside of Shrimp and make it look less unusual later

This works as intended for the shrimp milestone. I will create a new issue proposing a rework of the header navbar. Well done!
